LondonBritishLibrary/orient/BLorient11718A.xml

```diff
@@ -0,0 +1,226 @@
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">79</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>325</height>
+                                        <width>215</width>
+                                    </dimensions>
+                                </extent> 
+                                <foliation>The volume has no unified pagination, but each codicological unit has its own folio or page numbers.</foliation>
+                            </supportDesc>  
+                        </objectDesc>
+                        <bindingDesc>
+                            <binding xml:id="binding" contemporary="false">
+                                <decoNote xml:id="b1" type="Boards">European binding provided by the <placeName ref="INS00001BL"/>.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin><origDate notBefore="1923" notAfter="1924">1923/1924</origDate></origin>
+                        <provenance></provenance>
+                        <acquisition>On the verso of the last leaf - a note: 
+                            'Presented by <persName ref="PRS14446Douglas">Canon John A. Douglas</persName>. 
+                            <date when="1941">Jan. 1941</date>'</acquisition>
+                    </history>
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:Strelcyn1978BritishLibrary"/>
+                                            <citedRange unit="page">81</citedRange>
+                                        </bibl> 
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                    <msPart xml:id="p1">
+                        <msIdentifier/>
+                        <msContents>
+                            <summary/>
+                            <msItem xml:id="p1_i1">
+                                <locus from="2" to="82"/>
+                                <title>Answers given by Ethiopian Doctors to the questions addressed by Bishop Gwynne of Khartoum</title>
+                                <msItem xml:id="p1_i1.1">
+                                    <locus from="2" to="8"/>
+                                    <title>List of sixty questions</title>
+                                    <incipit xml:lang="am"><title>የ፷ጠያቂዎች፡ አርእስት።</title> 
```

Comment:
please check if it is not ጥያቄዎች

Reply:
It is like that in Strelcyns transcription:

<img width="304" alt="Screenshot 2024_Tayaqewoch" src="https://github.com/BetaMasaheft/Manuscripts/assets/40291787/280d51d1-3c7e-4a98-a78b-72802d690ff1">

Reply:
For sure, this is (the cataloguer's) mistake. ጠያቂ is just a different word from what is intended, ጠ instead of ጥ might have been a typo. -i read instead of -e is a typical mistake as -i and -e are more difficult to distinguish in some modern hands. Maybe just mark it `<sic>`.

Reply:
Do you mean, I should silently correct the ጠ to ጥ or mark the entire word with sic?